tests/integration/test_backups.py

```diff
@@ -29,6 +29,8 @@
     "failed to access/create the bucket, check your S3 settings"
 )
 FAILED_TO_INITIALIZE_STANZA_ERROR_MESSAGE = "failed to initialize stanza, check your S3 settings"
+CANNOT_RESTORE_PITR = "cannot restore PITR, juju debug-log for details"
+MOVE_RESTORED_CLUSTER_TO_ANOTHER_BUCKET = "Move restored cluster to another S3 bucket"
```

Comment:
I now saw this message blocking the test execution (the first test: `test_backup`).

Reply:
Fixed